### PR TITLE
Simplify prerequisites for running examples

### LINKED ISSUE #2014: Cannot run examples of ballista
**Describe the bug**
When I try to run ballista-examples/src/bin/ballista-sql.rs, I got an error: 

> thread 'main' panicked at 'failed to get arrow data dir: env `ARROW_TEST_DATA` is undefined or has empty value, and the pre-defined data dir `/Users/doki/.cargo/registry/src/github.com-1ecc6299db9ec823/arrow-10.0.0/../testing/data` not found

I find test data dir in df unit tests is:

>  datafusion::test_util::arrow_test_data()

in the example it's

> datafusion::arrow::util::test_util::arrow_test_data()

### PR DESCRIPTION
**Pull request by doki23**
# Which issue does this PR close?

Closes #2014

### CODE REVIEW

**Review by yjshen — APPROVED** [review]
LGTM.

File: ballista-examples/README.md [hunk 1]
```diff
@@ -47,9 +47,6 @@ RUST_LOG=info ./target/release/ballista-executor -c 4
 
 ## Running the examples
 
-Refer to the instructions in [DEVELOPERS.md](../DEVELOPERS.md) to define the `ARROW_TEST_DATA` and
```
Comment (xudong963) [inline ballista-examples/README.md:50]: https://github.com/apache/arrow-datafusion/blob/master/CONTRIBUTING.md#bootstrap-environment 
Can the relevant content in this file be deleted as well?

Comment (xudong963) [reply ballista-examples/README.md:50]: Btw, I found https://github.com/apache/arrow-datafusion/blob/master/DEVELOPERS.md doesn't exist 😂, maybe it refers to CONTRIBUTING.md

Comment (doki23) [reply ballista-examples/README.md:50]: > https://github.com/apache/arrow-datafusion/blob/master/CONTRIBUTING.md#bootstrap-environment Can the relevant content in this file be deleted as well?

yeah

**Review by xudong963 — APPROVED** [review]
Thank you @doki23

**Pull request merged**